linode_metadata/metadata_client.py

```diff
@@ -70,6 +78,8 @@ def __init__(
         self.base_url = base_url
         self._append_user_agent = user_agent
         self.timeout = timeout
+        self._debug = debug
+        logging.basicConfig(filename=debug_file, level=logging.DEBUG)
```

Comment:
```suggestion
        if debug:
            logging.basicConfig(filename=debug_file, level=logging.DEBUG)
```
I think we can only to config the logging when `debug` is enabled